solr/contrib/ltr/src/test-files/solr/collection1/conf/schema.xml

```diff
@@ -18,13 +18,22 @@
 
 <schema name="example" version="1.5">
   <fields>
-    <field name="id" type="string" indexed="true" stored="true" required="true" multiValued="false" />
+    <field name="id" type="string" indexed="true" stored="true" required="true" multiValued="false"/>
+    <field name="finalScore" type="string" indexed="true" stored="true" multiValued="false"/>
+    <field name="finalScoreFloat" type="float" indexed="true" stored="true" multiValued="false"/>
+    <field name="field" type="text_general" indexed="true" stored="false" multiValued="false"/>
     <field name="title" type="text_general" indexed="true" stored="true"/>
     <field name="description" type="text_general" indexed="true" stored="true"/>
     <field name="keywords" type="text_general" indexed="true" stored="true" multiValued="true"/>
     <field name="popularity" type="int" indexed="true" stored="true" />
```

Comment:
observation: via the `docValues="${solr.tests.numeric.dv}"` in the `<fieldType name="int" ...` and `popularity` being used in the `TestFieldValueFeature` we have some existing test coverage for doc values.

Reply:
Added a commit with additional `TestFieldValueFeature` coverage, for the numeric fields only so far i.e. `dvStrNumPopularity` and `dvStrBoolPopularity` not yet.